### PR TITLE
Tier marking

### PR DESCRIPTION
**Pull request by satyajitbulage**

### CODE REVIEW

**Comment by dajoRH** [comment]
Lint report for commit f8efaaa86204b6e12f4fc9e3a3b9c346349d26d3:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_instance_analysis.py [hunk 1]
```diff
@@ -21,6 +21,7 @@
 from utils.wait import wait_for
 from utils.blockers import GH, BZ
 
+pytestmark = [pytest.mark.tier(3)]
```
Comment (RonnyPfannschmidt) [inline cfme/tests/infrastructure/test_instance_analysis.py]: that one gets overridden by the next statement

**Comment by dajoRH** [comment]
Lint report for commit c151ac46889db08407e916ee6089f27fdef25024:

Everything seems all right :smile: .
*CFME QE Bot*

File: cfme/tests/infrastructure/test_instance_analysis.py (hunk 1 above)
Comment (satyajitbulage) [inline cfme/tests/infrastructure/test_instance_analysis.py]: @RonnyPfannschmidt I modified that statement. Kindly review it.

**Pull request merged**